.github/workflows/sbom.yaml

```diff
@@ -0,0 +1,42 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        required: true
+
+permissions: {}
+
+jobs:
+  test_bom_action:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    name: Install bom and generate SBOM
+    steps:
+      - name: Set tag name
+        shell: bash
+        run: |
+          TAG=${{ github.event.inputs.tag }}
+          echo "TAG=$TAG" >> "$GITHUB_ENV"
+      - name: Install bom
+        uses: kubernetes-sigs/release-actions/setup-bom@main
```

Comment:
Can we use the released version since we want to avoid going down the rabbit hole?

Reply:
PING @psschwei